cmake/Modules/FindComputeCpp.cmake

```diff
@@ -276,39 +370,49 @@ function(__build_spir targetName sourceFile binaryDir fileCounter)
             -isystem ${COMPUTECPP_INCLUDE_DIRECTORY}
             ${device_compiler_includes}
             -o ${outputSyclFile}
-            -c ${sourceFile}
-    DEPENDS ${sourceFile}
-    IMPLICIT_DEPENDS CXX ${sourceFile}
-    WORKING_DIRECTORY ${binaryDir}
+            -c ${SDK_BUILD_IR_SOURCE}
+    DEPENDS ${ir_dependencies}
+    IMPLICIT_DEPENDS CXX ${SDK_BUILD_IR_SOURCE}
```

Comment:
What is `CXX` here?

Reply:
https://cmake.org/cmake/help/v3.2/command/add_custom_command.html

It says "The language given specifies the programming language whose corresponding dependency scanner should be used." In this case, use the CXX rules.